samza-core/src/main/java/org/apache/samza/container/LocalityManager.java

```diff
@@ -45,32 +39,26 @@
   private final MetadataStore metadataStore;
 
   /**
-   * Builds the LocalityManager based upon {@link Config} and {@link MetricsRegistry}.
-   * Uses the {@link CoordinatorStreamValueSerde} to serialize messages before
-   * reading/writing into metadata store.
+   * <ul>
```

Comment:
Please format as a paragraph instead of a list everywhere. It's easier to read when looking at the code.

Reply:
Done.